x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_table/get_columns.tsx

```diff
@@ -207,5 +223,58 @@ export function getColumns({
       },
       sortable: true,
     },
+    {
+      width: '40px',
+      render: (instanceItem: PrimaryStatsServiceInstanceItem) => {
+        return (
+          <ActionMenu
+            id="instanceActionMenu"
+            closePopover={() =>
+              toggleRowActionMenu(instanceItem.serviceNodeName)
+            }
+            isOpen={itemIdToOpenActionMenuRowMap[instanceItem.serviceNodeName]}
+            anchorPosition="leftCenter"
+            button={
+              <EuiButtonEmpty
+                style={{ height: px(24) }}
+                onClick={() =>
+                  toggleRowActionMenu(instanceItem.serviceNodeName)
+                }
+              >
+                <EuiIcon type="boxesHorizontal" />
+              </EuiButtonEmpty>
```

Comment:
```suggestion
              <EuiButtonIcon
                iconType="boxesHorizontal"
                onClick={() =>
                  toggleRowActionMenu(instanceItem.serviceNodeName)
                }
              />
```

I think I've found what might be the cause of the different sizes of focus. I think the preferred way to have a icon button is to use the `EuiButtonIcon` component instead.

**Before**

<img width="102" alt="Screenshot 2021-04-19 at 10 25 18" src="https://user-images.githubusercontent.com/4104278/115206073-83e4a500-a0fa-11eb-82c0-e8df78850b62.png">

**After**

<img width="118" alt="Screenshot 2021-04-19 at 10 31 51" src="https://user-images.githubusercontent.com/4104278/115206049-7c250080-a0fa-11eb-82e8-46c278331df7.png">

Reply:
Thanks, I didn't know about the `EuiButtonIcon` I always use `EuiEmptyButton`.